flink-cdc-composer/src/main/java/org/apache/flink/cdc/composer/definition/TransformDef.java

```diff
@@ -72,16 +72,16 @@ public String getSourceTable() {
         return sourceTable;
     }
 
-    public Optional<String> getProjection() {
-        return Optional.ofNullable(projection);
+    public String getProjection() {
+        return projection;
     }
 
     public boolean isValidProjection() {
         return !StringUtils.isNullOrWhitespaceOnly(projection);
     }
 
-    public Optional<String> getFilter() {
-        return Optional.ofNullable(filter);
+    public String getFilter() {
+        return filter;
```

Comment:
CMIIW, This class has no accessibility annotation... so it's an `@Internal` class and breaking changes should be fine.

Maybe we can also tweak `getDescription` method to return nullable `String` for consistency?

Reply:
done~
ditto